### PR TITLE
docs: fix example for `examples` in the payload schema

### PR DESCRIPTION
**Pull request by aborg0**
---
title: "Fixes the example for `examples`"
---

---

**Related issue(s):**
#698 
---

### CODE REVIEW

File: spec/asyncapi.md [hunk 1]
```diff
@@ -1822,7 +1824,8 @@ properties:
     type: string
 required:
 - name
-example:
+examples:
+-
   name: Puma
   id: 1
```
Comment (magicmatatjahu) [inline spec/asyncapi.md:1830]: Please add `-` and first field in this same level (line):

```suggestion
- name: Puma
  id: 1
```

Comment (aborg0) [reply spec/asyncapi.md:1830]: The YAML spec suggests otherwise: https://yaml.org/spec/1.2.2/#example-sequence-of-mappings

Comment (magicmatatjahu) [reply spec/asyncapi.md:1830]: And in the next examples you have this :)
image

**Review by derberg — APPROVED** [review]
@magicmatatjahu thanks for the ping, I missed this one completely

@aborg0 thanks, sorry it took so long 😞

**Comment by fmvilas** [comment]
Forget about the last comment saying it was released in version 2.5.0-next-major-spec.1. I made a mistake and it created this version but it should actually be `3.0.0-next-major-spec.1`. There's a notice in the [release](https://github.com/asyncapi/spec/releases/tag/v2.5.0-next-major-spec.1). Apologies for the noise.

**Comment by derberg** [comment]
Recent comments about the release from the bot were added by mistake. More details in https://github.com/asyncapi/spec/issues/899

**Pull request merged**